### PR TITLE
Create the doc structure for v0.9

### PR DESCRIPTION
**Pull request by toddboom**

### CODE REVIEW

**Comment by toddboom** [comment]
@otoolep Here you go - this has all of the structure in place, using v0.8 as a starting point.

**Comment by otoolep** [comment]
Thanks very much, @toddboom .

**Comment by otoolep** [comment]
Starting to work on this @toddboom -- I am going to push more changes to this branch, so won't be merging for a little while.

File: source/docs/v0.9/concepts/key_concepts.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Key Concepts
+
+To understand InfluxDB, it is necessary to understand some key concepts. These concepts are outlined below.
+
+## Storing Data
+
+A **database** is quite similar to that of a traditional relational database, and is an organized collection of time-series data and retention policies. User privileges are also set on a per-database level.
+
+A **retention policies** is a logical namespace which maps to one or more shards, and has a _split factor_ and _replication factor_, both of which must be at least 1. An integral part of a retention policy is the _retention period_ – the time after which data is automatically deleted within the InfluxDB system. Every database has at least one retention policy.
```
Comment (pauldix) [inline source/docs/v0.9/concepts/key_concepts.md]: Split isn't a user facing concept anymore. Also, you should state first that `databases` have many `retention policies`, which have many `shards`. That is, retention policies belong to a database and shards belong to retention policies.

Then you have to define what a shard is.

File: source/docs/v0.9/concepts/key_concepts.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+**Split** is a characteristic of a retention policy and determines how many shards comprise that retention policy. Split factor is how the write throughput of a retention policy is scaled horizontally. It is orthogonal to the _Replication Factor_. **Replication factor** determines how many copies of data within retention policy exist on an InfluxDB cluster. If only a single copy exists, the replication factor is 1.
```
Comment (pauldix) [inline source/docs/v0.9/concepts/key_concepts.md]: Split isn't a user facing concept, but this is off. Split determines how many shards are created for each block of time. If a retention policy is for 30 days and we create a new set of shards per day, and we internally have split set to 3, then we'd always have at least 90 shards in the retention policy. At least, once it had been running for 90 days.

File: source/docs/v0.9/concepts/key_concepts.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Time-Series Data
+
+InfluxDB uses particular terms to describe the various components of time-series data, and the techniques used to categorize that data.
+
+A **series** is defined as a set of time-stamped data points, which includes a _measurement_, _tags_ and _fields_.
```
Comment (pauldix) [inline source/docs/v0.9/concepts/key_concepts.md]: It should also be noted that a series is defined by its `measurement + tag set`. Data points are really just a set of fields with a timestamp. The series is the combination of measurement and tags.

Comment (otoolep) [inline source/docs/v0.9/concepts/key_concepts.md]: "Key concepts" updated as required.

**Pull request merged**